sdk/cosmos/azure-cosmos/azure/cosmos/container.py

```diff
@@ -75,7 +95,49 @@ def _get_conflict_link(self, conflict_or_link):
             return u"{}/conflicts/{}".format(self.container_link, conflict_or_link)
         return conflict_or_link["_self"]
 
-    def get_item(
+    def read(
+        self,
+        session_token=None,
+        initial_headers=None,
+        populate_query_metrics=None,
+        populate_partition_key_range_statistics=None,
+        populate_quota_info=None,
+        request_options=None
+    ):
+        # type: (str, Dict[str, str], bool, bool, bool, Dict[str, Any]) -> Container
+        """ Read the container properties
+
+        :param session_token: Token for use with Session consistency.
+        :param initial_headers: Initial headers to be sent as part of the request.
+        :param populate_query_metrics: Enable returning query metrics in response headers.
```

Comment:
This would be a good place to hint at how to use the diagnostics callback  to see the requested information.

Reply:
That's going in a separate PR